examples/multiple-choice/index.es

```diff
@@ -0,0 +1,77 @@
+Element `multiple-choice`
+
+(class extends HTMLElement {
+
+  initialize () {
+    this.context.current = 0
+
+    this.context.questions_and_answers =
+      [
+        {
+          question: 'what popular tool for developing the client side application should we use?',
+          answers: ['React', 'Ember.js', 'Vue', 'Angular', 'Bootstrap', 'jQuery'],
+          // answer: undefined // There is no answer hehe
+        },
+
+        {
+          question: 'What city is known as "The Big 🍎 Apple ?"',
+          answers: ['Seattle, WA', 'Skokie, IL', 'New York City'],
+          answer: 2
+        }
+      ]
+  }
+
+  onidle () {
+    const
```

Comment:
this gets into @snuggs personal ES style preferences, which I think need a style guide to document sooner rather than later to minimize the amount of repetitive discussion on these topics.... basically you're always free to code your own style but @snuggs is prescribing a particular snuggsi set of style conventions because he likes how it looks/reads.